sklearn/tests/test_pipeline.py

```diff
@@ -342,8 +342,8 @@ def test_pipeline_score_samples_pca_lof():
     pca = PCA(svd_solver='full', n_components='mle', whiten=True)
     pipe = Pipeline([('pca', pca), ('lof', clf)])
     pipe.fit(X)
-    assert_equal(list(pipe.score_samples(X[:2])),
-                 [-0.9564705258081416, -1.0371487183896932])
+    assert_array_almost_equal(pipe.score_samples(X[:2]),
```

Comment:
assert_allclose is the preferred helper

Reply:
Done.

Reply:
`assert_array_almost_equal` is still being used.

Reply:
Sorry. I did something when rebasing the branch with master. I lost two commits along the way (rm assert_array_almost_equal and update what's new). Now it should be ok.